<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial</class>
 <widget class="QDialog" name="serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>260</width>
     <height>72</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_uart_open">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_return">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>校验位:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_uart_send">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_uart_num">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ttySAC0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ttySAC1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ttySAC2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_baud">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string>115200</string>
   </property>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_data_bit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string>8</string>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_parity">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>O</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_stop_bit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>流控制:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_flow_ctrl">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NONE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HARD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SOFT</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
